pages/integrations/git.md.erb

```diff
@@ -0,0 +1,92 @@
+# Git Server-Side Hooks
+
+You can trigger builds using Git hook scripts and the Buildkite REST API.
+
+This page explains how to trigger builds when you push to a Git server.
```

Comment:
> page

I suppose this could also be a "topic" or "guide" or something else. I'm not sure what's conventional around here.